src/Sylius/Behat/Resources/config/suites.yml

```diff
@@ -45,7 +45,7 @@ imports:
 #    - suites/api/promotion/applying_catalog_promotions.yml
```

Comment:
It should be uncommented

Reply:
`applying_catalog_promotions` requires endpoints from both product and productVariant(e.g. create), so I've added TODO to enable this after creating product and productVariant api resources